python/ray/tune/result_grid.py

```diff
@@ -190,6 +209,12 @@ def get_dataframe(
                     filter_metric="accuracy", filter_mode="max"
                 )
 
+            .. testoutput::
+                :hide:
+                :options: +SKIP
```

Comment:
There was an error in the contributor guide: https://github.com/ray-project/ray/pull/35462
```suggestion
                :options: +ELLIPSIS
```